build_tools/github/build_minimal_windows_image.sh

```diff
@@ -22,10 +22,21 @@ cp $WHEEL_PATH $WHEEL_NAME
 # Dot the Python version for identyfing the base Docker image
 PYTHON_VERSION=$(echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1:2})
 
+# TODO: Remove when 3.11 images will be available for
+# windows
+# scipy
+# pandas
+if [[ "$PYTHON_VERSION" == "3.11" ]]; then
+    PYTHON_VERSION=$(echo ${PYTHON_VERSION}-rc)
+    CIBW_TEST_REQUIRES="pytest threadpoolctl"
+    python -m pip install scipy==1.10.0.dev0
+fi
```

Comment:
Is this correct? Also, what is restrictive regarding pandas and Windows, here? 
```suggestion
# TODO: Remove when 3.11 images will are available on the
# default index for SciPy, pandas, Windows.
if [[ "$PYTHON_VERSION" == "3.11" ]]; then
    PYTHON_VERSION=$(echo ${PYTHON_VERSION}-rc)
    CIBW_TEST_REQUIRES="pytest threadpoolctl"
    python -m pip install scipy==1.10.0.dev0
fi
```

Reply:
I Have clarified the reasons of the restrictions: the windows docker image is tagged 3.11-rc, pandas is not available as wheel for 3.11 (and cannot be compiled on the minimal image for windows because the windows compiler is not installed, on purpose), scipy is from the custom nightly anaconda index.

Reply:
> pandas is not available as wheel for 3.11

Pandas 1.5.0 has Python 3.11 wheels, or are those not usable? https://pypi.org/project/pandas/1.5.0/#files

Reply:
Indeed, since September 22.
Thanks @EwoutH ! I have removed the limitation for pandas.